esbuild.js

```diff
@@ -22,6 +22,7 @@ const options = {
   target: ['node18'],
   minify: !dev,
   sourcemap: dev,
+  loader: { '.node': 'empty' },
```

Comment:
Set this to `file` so that the NAPI module gets copied to the output directory.

```suggestion
  loader: { '.node': 'file' },
```